src/Containers/MarketListCard/MarketListCard.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import './MarketListCard.scss';
+import images from '../../images/images';
+import { createMarketVendorLink, deleteMarketVendorLink, getAllMarketVendors } from '../../apiCalls';
+
+export const MarketListCard = ({ vendorId, name, id, marketsLinked, setMarketsLinked, vendorDescription }) => {
+
+  const handleAddSubmit = () => {
+    const vendorIdInt = parseInt(vendorId);
+    const marketIdInt = parseInt(id);
+    createMarketVendorLink(marketIdInt, vendorIdInt)
+      .then(data => {
+        const marketLinked = {id, name};
+        setMarketsLinked([...marketsLinked, marketLinked])
+      })
+      .catch(error => console.log(error))
+  }
+
+  const handleDeleteSubmit = () => {
+    const marketIdInt = parseInt(id);
```

Comment:
This was a great way to get the ID that connects Vendors to its specific Market.